portfolio/src/main/webapp/script.js

```diff
@@ -43,32 +43,60 @@ function addRandomQuote() {
     });
 }
 
-/** Adds an HTML element from Servlet. */
+/** Fetches comments from servlet and adds in DOM. */
 function showComments() {
-  const commentContainer = document.getElementById('comments');
-  commentContainer.innerHTML = '';
-
-  fetch('/data')
+  const maxComment = sessionStorage.getItem('max-comment') || 1;
+  document.getElementById("max-comment").value = maxComment;
+  document.getElementById('comments').innerHTML = '';
+  
+  fetch('/data?max-comment=' + maxComment)
     .then(response => response.json())
     .then(comments => {
       comments.forEach(comment => {
-        commentContainer.appendChild(createListElement(comment));
+        renderListComments(comment.propertyMap);
       });
     })
     .catch(error => void console.error(error));
 }
 
+/** Resets session storage value and shows that number of comments. */
+function refreshComments() {
+  const maxComment = document.getElementById("max-comment").value;
+  console.log(maxComment);
```

Comment:
Nit: remove debugging log or add some context to the message. Like `Refreshing comments with maximum ${maxComment}` or something.

Reply:
Removed the debugging log.